.travis.yml

```diff
@@ -6,50 +6,7 @@ before_install:
   - npm i -g grunt-cli
 jobs:
   include:
-    - stage: "Build and Test: Legacy"
-      name: "Build and Test: Legacy"
-      script:
-      - cd legacy/
-      - npm install
-      - grunt
-      - grunt test
-    - stage: "Build and Test: @microsoft/applicationinsights-web "
-      name: "Build and Test: @microsoft/applicationinsights-web "
-      script:
-      - set -e
-      - set NO_UPDATE_NOTIFIER=1
-
-      # - echo 'Checking for missing change logs...' && echo -en 'travis_fold:start:change\\r'
-      # - git fetch origin master:refs/remotes/origin/master -a
-      # - node common/scripts/install-run-rush.js change -v
-      # - echo -en 'travis_fold:end:change\\r'
-
-      - echo 'Checking for inconsistent dependency versions' && echo -en 'travis_fold:start:check\\r'
-      - node common/scripts/install-run-rush.js check
-      - echo -en 'travis_fold:end:check\\r'
-
-      - echo 'Installing...' && echo -en 'travis_fold:start:install\\r'
-      - rm -rf common/temp/npm-local
-
-      - node common/scripts/install-run-rush.js install
-      - echo -en 'travis_fold:end:install\\r'
-
-      - echo 'Building...' && echo -en 'travis_fold:start:build\\r'
-      - node common/scripts/install-run-rush.js rebuild --verbose
-      - echo -en 'travis_fold:end:build\\r'
-
-      - echo 'Running test...' && echo -en 'travis_fold:start:tests\\r'
-      - node common/scripts/install-run-rush.js test --verbose
-      - echo -en 'travis_fold:end:tests\\r'
-
-      - echo 'Running tslint...' && echo -en 'travis_fold:start:lint\\r'
-      - node common/scripts/install-run-rush.js lint --verbose
-      - echo -en 'travis_fold:end:lint\\r'
-    #- stage: "Build and Test: React Native Plugin"
-    #  name: "Build and Test: React Native Plugin"
-    #  script:
-    #  - cd extensions/applicationinsights-react-native
-    #  - npm install
-    #  - npm run build
-    #  - npm run test
-    #  - npm run lint
+    - script: ./.travis/test_rush.sh
+      name: "Tests"
+    - script: ./.travis/test_legacy.sh
+      name: "Legacy Tests"
```

Comment:
Why react and angular are not running?

Reply:
React Native & Angular tests aren't including in CI as of yet. React runs as part of `test_rush`.

Reply:
Added react to CI. @xiao-lix can add Angular in separate PR once it is ready